db/main.go

```diff
@@ -0,0 +1,52 @@
+package database
```

Comment:
A recommendation for package names, from [effective go](https://golang.org/doc/effective_go#package-names):

> Another convention is that the package name is the base name of its source directory; the package in src/encoding/base64 is imported as "encoding/base64" but has name base64, not encoding_base64 and not encodingBase64.

Or, in other words - `package db` to match the folder, or switch the folder to `database`. I generally recommend towards the latter - because this will be used far from source. "Far from source" being a commentary generally used in variables, where the idea is that the farther from definition a variable is used, the more verbose it should be. E.G. a for loop using `i` is fine, but if you planned to use an index variable two methods deep, having it be `index` would be more readable.

Reply:
Sounds good, will rename the directory to `database`.